sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/ServiceBusReceivedMessage.java

```diff
@@ -46,12 +47,14 @@
  */
 public final class ServiceBusReceivedMessage {
     private final ClientLogger logger = new ClientLogger(ServiceBusReceivedMessage.class);
-
     private final AmqpAnnotatedMessage amqpAnnotatedMessage;
+
     private UUID lockToken;
     private boolean isSettled = false;
     private Context context;
 
+    static final String SERVICE_BUS_MESSAGE_STATE_KEY = "x-opt-message-state";
```

Comment:
nit; should this be private instead of package private?

Reply:
nvm, I see we need it for testing.

Reply:
At the moment, we've shipped azure-core-amqp. I've set March milestone to move this to align with the other values. https://github.com/Azure/azure-sdk-for-java/issues/26898